2024-11-07 17:48:13.019 [main] ERROR org.solvd.atm.implementations.data.SessionInfoDAO - Unrecognized field "transaction" (class org.solvd.atm.domain.files.SessionInfo), not marked as ignorable (3 known properties: "id", "transactions", "atm_serie"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 5, column: 25] (through reference chain: org.solvd.atm.domain.files.SessionIndexesInfo["sessions"]->java.lang.Object[][0]->org.solvd.atm.domain.files.SessionInfo["transaction"])
2024-11-07 17:49:04.836 [main] ERROR org.solvd.atm.implementations.data.SessionInfoDAO - Unrecognized field "transaction" (class org.solvd.atm.domain.files.SessionInfo), not marked as ignorable (3 known properties: "id", "transactions", "atm_serie"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 5, column: 25] (through reference chain: org.solvd.atm.domain.files.SessionIndexesInfo["sessions"]->java.lang.Object[][0]->org.solvd.atm.domain.files.SessionInfo["transaction"])
2024-11-07 17:50:04.842 [main] ERROR org.solvd.atm.implementations.data.SessionInfoDAO - Unrecognized field "transaction" (class org.solvd.atm.domain.files.SessionInfo), not marked as ignorable (3 known properties: "id", "transactions", "atm_serie"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 5, column: 25] (through reference chain: org.solvd.atm.domain.files.SessionIndexesInfo["sessions"]->java.lang.Object[][0]->org.solvd.atm.domain.files.SessionInfo["transaction"])
2024-11-07 17:54:03.321 [main] ERROR org.solvd.atm.implementations.data.SessionInfoDAO - Error: can not add a new transfer to the session
2024-11-07 17:55:22.735 [main] ERROR org.solvd.atm.implementations.data.SessionInfoDAO - Unrecognized field "type" (class org.solvd.atm.domain.files.TransferInfo), not marked as ignorable (5 known properties: "amount", "currency", "referenceNumber", "origin", "destination"])
 at [Source: REDACTED (`StreamReadFeature.INCLUDE_SOURCE_IN_LOCATION` disabled); line: 11, column: 20] (through reference chain: org.solvd.atm.domain.files.SessionIndexesInfo["sessions"]->java.lang.Object[][0]->org.solvd.atm.domain.files.SessionInfo["transactions"]->java.lang.Object[][0]->org.solvd.atm.domain.files.TransferInfo["type"])
2024-11-07 20:59:03.655 [main] ERROR org.solvd.atm.implementations.data.ATMInfoDAO - src\main\resources\json\atmInfoBefore.json (The system cannot find the file specified)
2024-11-07 20:59:17.300 [main] ERROR org.solvd.atm.implementations.data.ATMInfoDAO - src\main\resources\json\atmInfoBefore.json (The system cannot find the file specified)
2024-11-07 21:17:00.600 [main] ERROR org.solvd.atm.utils.database.implementations.HikariCPDataSource - maxPoolSize cannot be less than 1
2024-11-07 21:17:18.944 [main] ERROR org.solvd.atm.utils.database.implementations.HikariCPDataSource - dataSource or dataSourceClassName or jdbcUrl is required.
2024-11-07 21:18:51.728 [main] INFO  org.solvd.atm.utils.database.implementations.HikariCPDataSource - Connection Created
2024-11-07 22:05:53.790 [main] ERROR org.solvd.atm.implementations.data.ATMInfoDAO - Error: there are no ATMs to update
2024-11-07 22:53:26.713 [main] ERROR org.solvd.atm.implementations.data.ATMInfoDAO - No serializer found for class org.solvd.atm.domain.stadistics.TotalTransactions and no properties discovered to create BeanSerializer (to avoid exception, disable SerializationFeature.FAIL_ON_EMPTY_BEANS) (through reference chain: org.solvd.atm.domain.files.ATMInfo[0]->org.solvd.atm.domain.files.ATMInfo["stadistics"]->org.solvd.atm.domain.files.ATMStadisticsInfo["transaction_counter"])
2024-11-07 23:12:56.012 [main] INFO  org.solvd.atm.utils.database.implementations.HikariCPDataSource - Connection Created

